lib/graphql/execution/interpreter/runtime.rb

```diff
@@ -513,13 +523,18 @@ def evaluate_selection_with_args(arguments, field_defn, next_path, ast_node, fie
               rescue GraphQL::ExecutionError => err
                 err
               rescue StandardError => err
-                begin
-                  query.handle_or_reraise(err)
-                rescue GraphQL::ExecutionError => ex_err
-                  ex_err
+                if selection_result.has_graphql_graph_parent_that_skips_list_items_that_raise
```

Comment:
API design question: before skipping an item, should we called the users `rescue_from` callback? (or perhaps make a new callback specific to item skipping?)

This `if` statement skips it for now, and silently skips the item. This isn't ideal, because the service owner wouldn't know items are being skipped. (e.g. if `rescue_from` reports to a bug tracker, it won't be called).